openapi/src/main/scala/endpoints/delegate/DocumentedRequests.scala

```diff
@@ -0,0 +1,34 @@
+package endpoints
+package delegate
```

Comment:
I put everything in the `openapi` project: the algebra, the `OpenAPI` interpreter and the delegate interpreters. Alternatively, we could have 3 distinct projects because the algebra doesn’t depend on the interpreters, but I thought that getting everything in a single dependency could make things simpler for users. Also, this dependency is quite cheap because it doesn’t depend on an heavy third-party `OpenAPI` library (in contrast to `circe` modules, which pull circe and cats, and thus justify to be in separate projects). Last, a good reason to separate interfaces and interpreters would be if we wanted to have other interpreters than the `OpenAPI` one (e.g. RAML), however the `openapi.DocumentedEndpoints` algebra is quite specific to the OpenAPI specification, so if we want to support alternative formats we would probably just define another algebra interface.